VideoWeb/VideoWeb/Services/AppRoleService.cs

```diff
@@ -49,14 +49,20 @@ public async Task<List<Claim>> GetClaimsForUserAsync(string uniqueId, string use
                     var typedException = ex as BookingsApiException<ProblemDetails>;
                     _logger.LogWarning(typedException, "User {Username} not found as a JusticeUser in BookingsApi", username);
                 }
-                return new List<Claim>();
             }
 
-            if (user == null) return new List<Claim>();
-            claims = MapUserRoleToAppRole(user.UserRoleId);
-            claims.Add(new Claim(ClaimTypes.GivenName, user.FirstName));
-            claims.Add(new Claim(ClaimTypes.Surname, user.Lastname));
-            claims.Add(new Claim(ClaimTypes.Name, user.FullName));
+            if (user == null)
+            {
+                claims = new List<Claim>();
+            }
+            else
+            {
+                claims = MapUserRoleToAppRole(user.UserRoleId);
```

Comment:
userRoleId doesnt exist anymore, it's an array of role object, if you update the booking.client - i have a pr to fix this https://github.com/hmcts/vh-video-web/pull/2018/files but if it makes more sense, can do it in this change  and ill close my pr. Up to you

Reply:
Thanks, I ended up copying the implementation from Admin Web